core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -1419,6 +1419,8 @@ wasm_instantiate(WASMModule *module, bool is_sub_inst, uint32 stack_size,
     module_inst->e =
         (WASMModuleInstanceExtra *)((uint8 *)module_inst + extra_info_offset);
 
+    module_inst->e->running_mode = 0;
```

Comment:
Had better `wasm_runtime_set_running_mode(module_inst, wasm_runtime_get_default_running_mode())`, which resolves the running mode 0 to actual running mode.